controllers/plantController.js

```diff
@@ -0,0 +1,18 @@
+const { catchAsync } = require("../utils/error");
+
+const plantService = require("../services/plantService");
+
+const getPlantInfo = catchAsync(async (req, res) => {
+  const { plantId } = req.params;
+  if (!plantId) {
+    throw new Error("KEY_ERROR");
+  }
+
+  const PlantInfo = await plantService.getPlantInfo(plantId);
```

Comment:
```suggestion
  const plantInfo = await plantService.getPlantInfo(plantId);
```
javascript에서는 class, enum, 변하지않는 상수데이터 등이 아니면 대문자로 시작하는 naming을 지양하는것이 컨벤션입니다.